src/neuroconv/datainterfaces/behavior/deeplabcut/_dlc_utils.py

```diff
@@ -119,30 +114,35 @@ def _ensure_individuals_in_header(df, dummy_name):
     return df
 
 
-def _get_pes_args(config_file, h5file, individual_name, infer_timestamps=True):
-    if "DLC" not in h5file or not h5file.endswith(".h5"):
+def _get_pes_args(config_file, h5file, individual_name, timestamps_available=False, infer_timestamps=True):
```

Comment:
```suggestion
def _get_pes_args(*, config_file: Path, h5file: Path, individual_name: str, timestamps_available: bool =False, infer_timestamps: bool =True):
```

While we're here, let's add a few things

- keyowrd-only arguments
- annotation typing

What do you think?

A docstring would also be nice (numpy style)